templates/press-centre/index.html

```diff
@@ -4,103 +4,62 @@
 
 {% block meta_description %}The home of Canonical media resources, news stories and press releases.{% endblock %}
 
+{% set is_paper = True %}
+
 {% block content %}
-  <section class="p-strip--standard-dark is-dark is-shallow">
-    <div class="row u-equal-height u-vertically-center">
-      <div class="col-7 p-card--suru">
-        <h1 class="p-heading--1">Press centre</h1>
-        <p class="p-heading--4">The home of Canonical media resources, news stories and press releases. Journalists seeking further information can contact <a class="p-link--inverted" href="mailto:pr@canonical.com">pr@canonical.com</a>
-        </p>
+
+<section class="p-section">
+  <div class="row">
+    <div class="col-9 col-medium-4 col-start-large-4 col-start-medium-3">
+      <div class="p-section">
+        <h1 class="u-no-margin--bottom">Press centre</h1>
+        <h2>All the latest news and featured stories <br class="u-hide--small u-hide--medium">from Canonical.</h2>
```

Comment:
@gloriaquintanilla has included it in the copydoc, I'll leave it unless told otherwise by a content person:
![image](https://github.com/canonical/canonical.com/assets/2741678/56690f2b-5917-4c6e-8075-3af2d324777e)

Reply:
Sure. It'd be helpful to link or mention the doc in your pr description if we're meant to reference it during QA. I was [told originally](https://warthogs.atlassian.net/browse/WD-3744?focusedCommentId=257566)  that we should be treating design as the source of truth for copy for now as the docs are not finalized. If that's no longer the case then whoever's reviewing work should be made aware

Reply:
I think in this case they match, don't they?  I'm following the copy doc myself when creating the design.